spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/AbstractStreamService.java

```diff
@@ -154,6 +168,14 @@ else if (DeploymentState.deploying == status) {
 			throw new StreamAlreadyDeployingException(name);
 		}
 		doDeployStream(streamDefinition, deploymentProperties);
+
+		final Map<String, Object> auditedData = new HashMap<>(2);
+		auditedData.put("streamDefinitionDslText", streamDefinition.getDslText());
```

Comment:
We can use constant names for these fields as they could be useful for any other systems that are built to use the audit data.

Reply:
:+1: